### ISSUE TITLE
[18-Mottaker] Motta store filer

### ISSUE THREAD

**Issue by erikhag1** [Altinn/altinn-broker#77]
## Users view

Title: **Motta store filer**
User role(s): Mottaker
Users value statement(s): 
TBD

## Vendors view

High level features (capabilities):
 * Formidle oversendendelse av store filer fra en avsender til en eller flere mottakere

```[tasklist]
### Features 
- [ ] GUI for mottaker
- [ ] Støtte for filstørrelser over 100GB
```

```[tasklist]
### Work items

```

## Item attributes

_Note: Automatically updated properties, not intended for change by you;)_

_Issue type: epic_
_Concept no: 18_
_Stage: Operation_
_ArchiConceptID: id-3ff45cacf4ec4b1da1e8edf64714c4a3_
_GithubIssueID: [to be autofilled]_

**Comment by erikhag1** [comment]
Løsning fpr GUI: Felles arbeidsflatre

**Comment by ToreMasoy** [comment]
Ja, det skal være mulig å sende og motta store filer, men er det egentlig noen forskjell på funksjonaliteten fra mindre filer? Hvor går i så fall grensen mellom små og store filer?
